src/thunderbots/software/gui/drawing/draw_functions.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <QtWidgets/QGraphicsScene>
+#include <functional>
+
+// An abstract base class of a DrawFunction that is used to draw objects
```

Comment:
"An abstract base class of a DrawFunction"? Phrasing is a bit confusing here. How about: "This class is used to ..."

Reply:
In general we should probably have a more detailed comment here.

Reply:
fixed